react/interfaces/custom-pixel-event.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface CustomPixelEventInterface {
```

Comment:
I guess this file is not needed anymore

Reply:
Also, this interface can be written like this

```
export type CustomPixelEventInterface = Record<string, CustomPixelEventPropTypes | CustomPixelEventPropTypes[]>
```

Reply:
@victorhmp @igorbrasileiro yes, this was the old implementation, i`ve removed this file